### DISCUSSION TITLE
Q1S1 '23 Prioritisation

### DISCUSSION THREAD

**Discussion by BenJam** [opencollective/opencollective#6228] [General]
_This discussion is designed to reach a consensus on the areas of focus on in the first six weeks of 2023_

At our retreat in September, I spoke about our issues trying to maintain a process in which design and product support the rest of the team to develop ideas for prioritisation, a process that has wound up looking like this:
<p align="center">
<img src="https://user-images.githubusercontent.com/158833/206460860-26143351-8683-4963-b51d-648b3ce7bfb8.png" width="350">
</p>

Last quarter we asked everyone to vote on 30 individual ideas to prioritise, asking each of you to read, understand and compare ideas in order to give an informed opinion. I think it's fair to say that's a big request to make of everyone.

Since the retreat @iamronen and I have been thinking about how we move from supporting you all in pitching your ideas individually toward a process where your ideas are built into a more coherent, cohesive, and considered _strategy_ for the platform that we can all have an informed say in. 

## Product Areas

To enable us to do this, we broke Open Collective into a number of product areas that describe parts of the platform:

| Area | Includes |
| --- | --- |
| Core | Infrastructure: stack, Hosting, Deployment & Scaling, Tooling, Frontend App, Testing & Documentation, Performance & Monitoring, System Security, Language, Accessibility | 
| Notifications & Attention Management | Email, In-app, batching | 
| Accounting | Ledger, Payment processing (Stripe, Wise, Paypal, etc.) , Host and Platform Fee, Splits, Currencies |
| Inbound Integrations | API, OAuth | 
| Discovery | Search. Explore, Marketing, Horizons |
| Onboarding | Individuals, Collectives, Funders, Hosts | 
| Fundraising | Contributions, Tiers, Goals, Projects | 
| Contributions | Incoming Funds Allocation, Refunds & Dispute Resolution, Pending Contributions | 
| Expenses | Invoices, Reimbursements., Virtual Cards. Recurring Payments, Expense Submission & Authorization, Invited Expenses |
| Presence and Identity | Profiles (User, Organisation & Collective), Contributions, Payment Methods, Expenses | 
| Governance & Fiscal Responsibility | Budget Management . Funding Distribution & Management, Impact (inc budget visualisation), Auditing & Accounting, Fraud Protection | 
| Support and Ops | root actions, Freshdesk integration, payment management and processing  |
| Social and Community | Events, Updates, Conversations | 
| Partnerships | GitHub, CoBudget, 

This 'map' of Open Collective isn't intended to be comprehensive. It is merely a tool for us to decide where we should focus our attention and to develop the platform in a way that allows us to incorporate ideas while designing (and executing) a more consistent vision. 

## Where we spent our time in 2022

The map also allows us to see where we've invested our time over the course of the year, and to compare it to where we _asked_ to invest our time:

| Projects we prioritised for development | Projects we pitched for development |
| --- | --- | 
| ![Screenshot 2022-12-08 at 16 03 24](https://user-images.githubusercontent.com/158833/206497666-e1a1b36e-001b-4ca6-b629-10df7a00934c.png) | ![Screenshot 2022-12-08 at 16 15 53](https://user-images.githubusercontent.com/158833/206501894-da623efc-0f86-4f93-948c-c3e8c9098329.png) |

Already we can see that we've arguably over-investing in areas like accounting and under-invested in areas like presence and identity. @iamronen goes into detail [over on Loom](https://www.loom.com/share/9c3fe6dbea4644818c6384f7293c35c6) about product areas and this analysis. 

## Product Strategy

Finally, this map gives us a way to formalise a strategic vision for areas of the platform, to describe the Open Collective we want to build in a broad sense, to build excitement within the team and to align everyone on a path that we develop together. 

## Contributing to this discussion 

It will take time to develop a strategy for each of the areas that we've defined above, but we can focus our attention on the things that we think are most lacking _today_, which is where you come in. 

This survey allows you to rank each of the product areas above, from the most (top) to least (important) in your mind right now: 

https://x7rwr9qad3h.typeform.com/to/zEzYvOE9 

with the result of this survey we will begin _immediately_ working to develop the strategy for those areas, working to ensure that the top 2-3 areas are mature enough to begin work on them in 2023. 

## What about pitching ideas, reporting bugs etc?

Nothing changes in that respect; you're still free to [open an issue, bug or to pitch an idea](https://github.com/opencollective/opencollective/issues/new/choose). What changes is that instead of deciding together on individual projects, we decide on which areas of the platform deserve our attention, and we build your ideas into the vision for those areas. 

That said, this is still a discussion, you can still talk about why you think we should focus our time on a given area over another, reference specific issues and problems, and to pitch for specific ideas if you feel the need to. 

## Results

The results of the survey have been very small, with only 6 respondents. Of those responses people ranked support and ops #5906, presence and identity and core application infrastructure. 

<img width="569" alt="Screenshot 2023-01-04 at 17 02 03" src="https://user-images.githubusercontent.com/158833/211029368-ba7443b8-5009-41be-8621-e9b6cdc36534.png">

**Comment by kewitz** [comment]
This is a good improvement. This removes the chance of us prioritizing an issue from an area that we feel requires attention without a clear strategy for the area as a whole. Ideally, we would have a clear strategy for all areas and know exactly what are the next steps before deciding which areas we want to focus on.

**Comment by BenJam** [comment]
just noting that @carolineawoolard was talking about improving our onboarding and that previously we had been through improvement in this area https://github.com/opencollective/opencollective/issues/4829#issuecomment-1138111062 and that the next thing to look at was to get rid of the 'open source 'any collective' step and to add tags and location to the onboarding flow(s) followed by improvements (that we have designs for) to the 'join a host' part.

**Comment by BenJam** [comment]
As per [our strategy](https://blog.opencollective.com/oc-strategy-2023-2024/) we should also look at how we'll emphasise verified hosts over hosts that we have little involvement with. This will mostly be addressed by https://github.com/opencollective/opencollective/issues/4240 but there will be other places where we'll need to address concerns.
